src/bootstrap/test.rs

```diff
@@ -2011,8 +2020,8 @@ impl Step for Crate {
         }
 
         builder.info(&format!(
-            "{} {} stage{} ({} -> {})",
-            test_kind, krate, compiler.stage, &compiler.host, target
+            "{} {:?} stage{} ({} -> {})",
```

Comment:
Hm, is this going to be a really long array in the future when we're not testing one by one?

Today it ends up as `Testing ["panic_abort"] stage1 (x86_64-unknown-linux-gnu -> x86_64-unknown-linux-gnu)` (for example) which seems OK though not great. Let's leave it for now though.

Reply:
It can, yes:
```
Testing ["rustc-main", "rustc_data_structures", "rustc_metadata", "rustc_middle", "rustc_trait_selection"] stage0 (x86_64-unknown-linux-gnu -> x86_64-unknown-linux-gnu)
```
I think I remember you saying we're not going to encourage running all unit tests, though; I think it's ok to have this be kind of long if people do it anyway.